Second 1200+ Words Commit for Appendix C-Git-Commands

cherryloua opened the pull request:
Hello,

Please, take some time for review my second PR Request. The translation covers from `git difftool` command to fifth paragraph of `git log` command. 

Thanks,
cherryloua

cherryloua committed:
Update 1-git-commands.asc

cherryloua committed:
Update 1-git-commands.asc

cherryloua committed:
Update 1-git-commands.asc

cherryloua committed:
Update 1-git-commands.asc

severinolorillajr commented:
Hi @cherryloua, please find someone from our group who can review and proofread your work.
Then let me know as soon as you have this reviewed. 

Thanks.

mind-blast commented:
@cherryloua checking your translation. it seems @severinolorillajr is busy right now.. wait a moment.

mind-blast commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -102,8 +102,9 @@ Una nating na pag-usapang ang `status` sa << _ git_basics_chapter # _checking_st
+
 Ang kautosang `git diff` ay ginagamit kapag nais mong makita ang mga pagkakaiba sa pagitan ng dalawang trees.
-Ito ay maaaring pagkakaiba sa pagitan ng iyong gumaganang kaligiran at ang iyong staging na lugar (`git diff` mismo), sa pagitan ng iyong staging na lugar at ang iyong huling commit (` git diff --staged`), o sa pagitan ng dalawang mga commit (`git diff master branchB `).
+Ito ay maaaring pagkakaiba sa pagitan ng iyong gumaganang kaligiran at ang iyong staging na lugar (`git diff` mismo), sa pagitan ng iyong staging na lugar at ang iyong huling commit (` git diyff --staged`), o sa pagitan ng dalawang mga commit (`git diff master branchB `).
```
(` git diyff --staged`)   please change to (` git diff --staged`)

heywens replied:
Suggestions - please change the following words:
*  ``kautosang`` to ``kautusang``
*  ``kaligiran`` to ``environment`` 
hi @severinolorillajr, can we make this as one of the reserved words? or do we have to translate it? please advise.

mind-blast replied:
@heywens i think environment is better since it is a technical terms which really fits for git.

mind-blast commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -118,64 +118,64 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
+Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ganito niya ginagawa.
 
-Tiningnan namin kung paano mag-sign magawa ng cryptographically kasama ang `-S` flag sa <<_git_tools#_signing_commits>>.
+Tiningnan namin kung paano mag-sign ng mga commit na cryptographically kasama ang `-S` na palatandaan sa <<_git_tools#_signing_commits>>.
```
i think flag is a reserve word on  git.  please check   :    flag    and    na palatandaan

mind-blast commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -118,64 +118,64 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-Ang `git branch` command ay talagang isang bagay ng isang tool sa pamamahala ng sangay.
-Maaari itong ilista ang mga sangay na mayroon ka, lumikha ng isang bagong sangay, tanggalin ang mga sanga at palitan ang pangalan ng mga sanga.
+Ang `git branch` command ay talagang isang tool na naayon sa pamamahala ng sangay.
```
please also review also.  branch is a reserve word. please recheck.

mind-blast commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -206,11 +206,11 @@ Susubukan nito ang kasalukuyang sangay sa resulta ng pagsasama.
+Tinalakay namin kung paano gumawa ng isang squashed merge (kung saan ang Git ay nagpahiwatig na gumagana ang merge ngunit nagpapanggap lamang ito na tulad ng isang bagong commit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa pinakadulo ng << _ distributed_git # _public_project >>.
 
-Nagpunta kami ng maraming tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` command at ang` --abort` flag upang i-abort ang problema sa pagsasama sa << _ git_tools # _advanced_merging >>.
+Maramin din kaming tinalakay tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` na ustos at ang` --abort` na palatandaan upang i-abort ang problema sa pag-merge sa << _ git_tools # _advanced_merging >>.
```
please change Maramin   to Marami  .   na ustos  to na utos

mind-blast commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -222,19 +222,17 @@ Binanggit namin ito nang mabilis sa << _ git_branching # _basic_merge_conflicts
 
-In <<_distributed_git#_private_team>> and <<_git_tools#_commit_ranges>> we cover the `branchA..branchB` syntax to use the `git log` command to see what commits are unique to a branch relative to another branch.
-In <<_git_tools#_commit_ranges>> we go through this fairly extensively.
+Sa <<_distributed_git#_private_team>> at <<_git_tools#_commit_ranges>> aming tinalakay ang `branchA..branchB` na syntax para gamitin ang `git log` na utos para makita kung ano ang kaibahan sa isang katugong sangay sa iba pang sangay.
```
change aming   to aming .    sangay   to  branch ..

mind-blast commented:
Since i cannot edit it directly , please changed them according to what i suggest.  Thank you.

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 ==== git commit
 
-Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer sa kasalukuyang branch hanggang dito.
+Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer hangang sa kasalukuyang branch nito.
```
hangang to hanggang?

severinolorillajr replied:
Please update this as well. Thanks.

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-We first effectively cover the simplest use of `git reset` in <<_git_basics_chapter#_unstaging>>, where we use it to unstage a file we had run `git add` on.
-Una naming epektibong sinasakop ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang unstage isang file na kami ay tumakbo git idagdag sa.
+Una naming epektibong tinalakay ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang ma unstage isang file na pinatakbo ang `git add`.
```
Please be careful with this tags <<_git_basics_chapter#_unstaging>>
Don't remove them.

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-We then cover it in quite some detail in <<_git_tools#_git_reset>>, which is entirely devoted to explaining this command.
-Pagkatapos ay tinatakpan namin ito sa lubos na detalye sa _git_tools.asc, na lubos na nakatuon sa pagpapaliwanag sa utos na ito.
+Pagkatapos ay tinalakay namin ito ng lubos ang detalye sa _git_tools.asc, na nakalaan sa pagpapaliwanag sa utos na ito.
```
*namin to natin.
natin ito ng lubusan sa 
<<_git_tools#_git_reset>> Again please don't remove the tags.

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-We use `git reset --hard` to abort a merge in <<_git_tools#_abort_merge>>, where we also use `git merge --abort`, which is a bit of a wrapper for the `git reset` command.
-Ginagamit namin ang git reset - hard upang i-abort ang isang sumanib sa _git_tools.asc, kung saan ginagamit din namin ang git merge --abort, na kung saan ay isang bit ng wrapper para sa git reset command.
+Ginagamit namin ang git reset `--hard` upang i-abort ang pagsanib sa _git_tools.asc, kung saan ginagamit din namin ang `git merge --abort`, na kung saan ay isang kaunting pagbabalut para sa `git reset` command.
```
<<_git_tools#_abort_merge>>

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
-We first introduce it in <<_git_branching#_create_new_branch>> and we go through most of its other features (listing and deleting) in <<_git_branching#_branch_management>>.
-Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa command na 'branch` at ginagamit ito sa buong kabanata.
-Una naming ipakilala ito sa << _ git_branching # _create_new_branch >> at kami ay dumadaan sa karamihan ng iba pang mga tampok nito (listahan at pagtanggal) sa << _ git_branching # _branch_management >>.
+Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa utos na 'branch` at ginagamit ito sa buong kabanata.
```
<<_git_branching#_git_branching>>

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
-Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa command na 'branch` at ginagamit ito sa buong kabanata.
-Una naming ipakilala ito sa << _ git_branching # _create_new_branch >> at kami ay dumadaan sa karamihan ng iba pang mga tampok nito (listahan at pagtanggal) sa << _ git_branching # _branch_management >>.
+Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa utos na 'branch` at ginagamit ito sa buong kabanata.
+Una naming ipinakilala ito sa << _ git_branching # _create_new_branch >> at kami ay dumaan sa karamihan ng iba pang mga tampok nito (listahan at pagtanggal) sa << _ git_branching # _branch_management >>.
```
<< _ git_branching # _create_new_branch >>

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-Finally, we go into some implementation detail in <<_git_internals#_the_head>>.
-Sa wakas, pumunta kami sa ilang detalye ng pagpapatupad sa << _ git_internals # _the_head >>.
+Sa panghuli, pumunta kami sa ilang detalye ng pagpapatupad sa << _ git_internals # _the_head >>.
```
<<_git_internals#_the_head>>

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-We covered how to do a squashed merge (where Git merges the work but pretends like it's just a new commit without recording the history of the branch you're merging in) at the very end of <<_distributed_git#_public_project>>.
-Tinakpan namin kung paano gumawa ng isang nakakulong na pagsasama (kung saan ang Git ay nagpahiwatig ng trabaho ngunit nagpapanggap na tulad ng isang bagong komit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa dulo ng << _ distributed_git # _public_project >>.
+Tinalakay namin kung paano gumawa ng isang squashed merge (kung saan ang Git ay nagpahiwatig na gumagana ang merge ngunit nagpapanggap lamang ito na tulad ng isang bagong commit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa pinakadulo ng << _ distributed_git # _public_project >>.
```
<<_distributed_git#_public_project>>

severinolorillajr commented on book/C-git-commands/1-git-commands.asc:
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-We went over a lot about the merge process and command, including the `-Xignore-space-change` command and the `--abort` flag to abort a problem merge in <<_git_tools#_advanced_merging>>.
-Nagpunta kami ng maraming tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` command at ang` --abort` flag upang i-abort ang problema sa pagsasama sa << _ git_tools # _advanced_merging >>.
+Maramin din kaming tinalakay tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` na ustos at ang` --abort` na palatandaan upang i-abort ang problema sa pag-merge sa << _ git_tools # _advanced_merging >>.
```
<<_git_tools#_advanced_merging>>

severinolorillajr commented:
Hi @cherryloua , please be very careful with the tags,
for example <<_git_tools#_advanced_merging>>
don't remove the tags, or add spaces to it.
Just leave it as it is.
Thanks.

dranem commented:
change this one also << _ git_basics_chapter # _checking_status >> . Tip : View the original file so that you will have a guide for the original syntax.

cherryloua commented:
Thank you for the feedback guys.... I'm going to edit them now. Thanks again.

cherryloua committed:
Corrected the suggested changes from the PR Review

I really don't know what happened to the tags... maybe because I export it from the phone as a pdf and copy paste it to the PC. In case your wondering why I did that, I originally used my phone for the translation and I accidentally deleted all my progress and the latest good copy that I have is a backup of a text editor app. On top of that, if directly copy paste it from the editor there were weird characters appearing thus pdf was used. lol

Anyways, thank you for the review guys. 

P.S. From now on, I'll be working on a PC. haha

severinolorillajr commented:
One peniding review, then we can merge this.

cherryloua committed:
Changed "hangang" to "hanggang"
